InterfaceData/editorRequirements.txt

```diff
@@ -35,15 +36,15 @@
 147,Standing on right side
 155,If Ki Charged
 157,Rage
-176,In Max-Mode
+176,Geese in Max-Mode
 182,Near Wall
 217,Player character ID
 218,Player NOT character ID
 219,Opponent character ID
 220,Opponent NOT character ID
-221,Partner character ID
-222,Partner NOT character ID
-223,Gamemode
+221,Player character model
+222,Opponent character model  
```

Comment:
This and the one above should be kept as ID: those requirements are there to check characters' numeric identifier and mdoel does not convey that mean ing

Reply:
Also, why the change from "Patner"?

Reply:
For the TTT2 moveset, these requirements check the partner ID. But for Tekken 7 movesets, 221 checks the static character ID variable (the one in Player struct). If you pick Devil Jin and/or Kazuya and then change the value of that static variable, it will instantly disable their wing animations.
As for requirement 223, It is used to play character specific intros and outros(T7 movesets). Parameter is the ID of the opponent.
How does 221 differ from 217? The latter check the character ID inside the movelist while the former checks it in the Player struct.

Reply:
217 also checks for the ID inside the player struct, the fact that it is affected by the character ID is the movelist is my own doing

Reply:
Noted!